data/json/npcs/missiondef.json

```
@@ -287,6 +287,7 @@
     "id": "MISSION_GET_RELIC",
     "type": "mission_definition",
     "name": { "str": "Find Relic" },
+    "description": "Find a relic of a forgotten saint.  Places like a <color_yellow>cathedral or a museum</color> might have one.",
```

```suggestion
    "description": "Find a relic of a forgotten saint.  Places like <color_yellow>cathedrals</color> or <color_yellow>museums</color> might have one.",
```
I think I did the colors right on this one? But truth be told I've never done that before, so take this one with a grain of salt